demo/README.md

```diff
@@ -0,0 +1,14 @@
+Woodchipper Demo Application
+-----------
+
+Demo Application to display Woodchipper logging using Zappa and Flask in AWS to log out to Cloudwatch.
+
+- Requires Tackle DEV AWS Access to publish.
+- Run `zappa update dev` to publish
```

Comment:
If we do add this to the lib, we'll need to remove the reference to Tackle.

Reply:
Good call - sorry was thinking of this for Tackle. I'll remove that and the zappa file actually, and just put a blurb in that it can be tested with zappa